tests/chainer_tests/functions_tests/activation_tests/test_rrelu.py

```diff
@@ -3,82 +3,76 @@
 import numpy
 
 import chainer
-from chainer import backend
 from chainer import cuda
 from chainer import functions
-from chainer import gradient_check
 from chainer import testing
 from chainer.testing import attr
 
 
 @testing.parameterize(*testing.product({
     'train': [True, False],
-    'shape': [(3, 2), (5, 6)],
+    'shape': [(3, 2), ()],
     'dtype': [numpy.float16, numpy.float32, numpy.float64],
 }))
-class TestRReLU(unittest.TestCase):
-
-    def setUp(self):
-        # Avoid unstability of numerical grad
-        self.x = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
-        self.x[(-0.05 < self.x) & (self.x < 0.05)] = 0.5
-        self.gy = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
+@testing.inject_backend_tests(
```

Comment:
I think it's better to put `@testing.inject_backend_tests` before `@testing.parameterize`, because parameters are more closely related to the test logic.